ruby/lib/transformer/syskit/frame_propagation.rb

```diff
@@ -332,19 +299,8 @@ def propagate_task(task)
         #   frame to a global frame name
         def self.initial_frame_selection_from_device(task, dev)
             tr = task.model.transformer
-            if selected_frame = dev.frame
-                if !task.transformer.has_frame?(selected_frame)
-                    raise Transformer::InvalidConfiguration, "undefined frame #{selected_frame} selected as reference frame for #{dev}, known frames: #{task.transformer.frames.sort.join(", ")}"
-                end
-            end
-            if selected_transform = dev.frame_transform
-                if !task.transformer.has_frame?(selected_transform.from)
-                    raise Transformer::InvalidConfiguration, "undefined frame #{selected_transform.from} selected as 'from' frame for #{dev}, known frames: #{task.transformer.frames.sort.join(", ")}"
-                end
-                if !task.transformer.has_frame?(selected_transform.to)
-                    raise Transformer::InvalidConfiguration, "undefined frame #{selected_transform.to} selected as 'to' frame for #{dev}, known frames: #{task.transformer.frames.sort.join(", ")}"
-                end
-            end
+            selected_frame = dev.frame
+            selected_transform = dev.frame_transform
```

Comment:
With that change, we're losing the information that the frame selection was coming from the device (and not the task). Now way to keep it here-ish ?